contracts/CourtPresaleActivate.sol

```diff
@@ -62,6 +155,21 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
         bondedToken.approve(address(registry), bondedTokensObtained);
         registry.stakeFor(_from, bondedTokensObtained, abi.encodePacked(ACTIVATE_DATA));
 
+        // refund leftovers if any
+        _refund(bondedToken);
```

Comment:
This goes with the above question; how would someone end up with an amount of bonded tokens to need refunding? Is it only possible if someone accidentally sent the bonded token to this contract and then the next person using this contract will receive it?

Reply:
Yes, that, or a bug in the presale or registry contracts (that was my initial assumption, as with Uniswap).